### PR TITLE
Series for vada_eluthu

### PR DESCRIPTION
**Pull request by aswinvenk**
Added series for vada_eluthu

Please do check and let me know

### CODE REVIEW

File: Tamilcode.json [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+          "\u0B95\u0BCD\u0BB7\u0BC6",
+          "\u0B95\u0BCD\u0BB7\u0BC7",
+          "\u0B95\u0BCD\u0BB7\u0BC8",
+          "\u0B95\u0BCD\u0BB7\u0BCA",
+          "\u0B95\u0BCD\u0BB7\u0BCB",
+          "\u0B95\u0BCD\u0BB7\u0BCC"
+        ]
+      },
+      {
+        "shree": "\u0BB6\u0BCD\u0BB0\u0BC0"
```
Comment (PugazhendiN) [inline Tamilcode.json]: we have to keep this content as a list like other series to maintain Json uniformity.

Comment (aswinvenk) [reply Tamilcode.json]: This is the same structure as uyir mei eluthu

Comment (aswinvenk) [reply Tamilcode.json]: list of dict

**Comment by aswinvenk** [comment]
Problem solved

**Comment by aswinvenk** [comment]
If possible try changing the title to tamil eluthukal

**Pull request merged**